<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayerWindow</class>
 <widget class="QWidget" name="MusicPlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_AlbumCover">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>170</width>
     <height>170</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/Images/AlbumCover.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_MusicName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Music Name</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>720</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton_PlayPause">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>5</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton:hover
{ border-image: url(:/Buttons/Play.svg); }*/
QPushButton:pressed
{ border-image: url(:/Buttons/Play.svg); }
</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Buttons/Play.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Next">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/Next.svg); }</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Buttons/Next.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Previous">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/Previous.svg); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Buttons/Previous.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mute">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/Mute.svg); }</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Buttons/Mute.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_LeftTime">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>25</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/Stop.svg); }</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Buttons/Stop.svg</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_Volume">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>35</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Artist">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Artist Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Composer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Composer Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Background/MusicPlayerBackground.jpg);</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>720</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/Close.svg); }</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Buttons/Close.svg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PlaybackMode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenFile">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed
{ border-image: url(:/Buttons/OpenFile.svg); }</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Buttons/OpenFile.svg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="slider_Progress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Slider 外面的槽（外邊框），border: 0px設置為外面不顯示外邊框。 */
QSlider::groove:horizontal { 
border: 1px solid #4A708B;
background: #C0C0C0;
height: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
}

/* 滑鍵 */
QSlider::handle:horizontal 
{
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #45ADED, stop:0.778409 rgba(255, 255, 255, 255));

	width: 11px;
	margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}

/* 滑鍵滑過的區塊 */
QSlider::add-page:horizontal 
{
	background: #575757;
	border: 0px solid #777;
	height: 10px;
	border-radius: 2px;
}

/* 滑鍵未到達的區塊 */
QSlider::sub-page:horizontal 
{ 
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
	background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, stop: 0 #5DCCFF, stop: 1 #1874CD);
	border: 1px solid #4A708B;
	height: 10px;
	border-radius: 2px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="list_PlayList">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>label_AlbumCover</zorder>
  <zorder>label_MusicName</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_Artist</zorder>
  <zorder>label_Composer</zorder>
  <zorder>line</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>pushButton_PlaybackMode</zorder>
  <zorder>pushButton_OpenFile</zorder>
  <zorder>slider_Progress</zorder>
  <zorder>list_PlayList</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
